generic_permissions/visibilities.py

```diff
@@ -39,6 +39,25 @@
 filter_queryset_for = VisibilitiesConfig.decorator
 
 
+def _should_bypass_field(instance, field_name: str) -> bool:
+    bypass_fields_settings = (
+        getattr(settings, "GENERIC_PERMISSIONS_BYPASS_VISIBILITIES", {}) or {}
+    )
+    meta = getattr(instance, "_meta", None) or getattr(instance, "Meta", None)
+    app_label = getattr(meta, "app_label", "")
+    print("meta", meta)
+    print("app_label", meta)
+    bypass_fields = bypass_fields_settings.get(
+        f"{app_label}.{type(instance).__name__}",
+        [],
+    )
+
+    if bypass_fields == "__all__":
+        return True
```

Comment:
I'm not sure about this. It's a nice shortcut, but given that this is security-related, I'd rather have every property listed explicitly

Reply:
This has been requested by me as we'll need this for testing in alexandria. Let's keep it please.

Reply:
Fair play, but then I think we'll need a test for that :-)

Reply:
And a big warning in the README please “DO NOT USE IN PRODUCTION, insecure!”

Reply:
Nah, it's not that extreme, I'd suggest a warning along the lines of "Careful with this, as it disables visibility checks in *all* of the relationships for this model. Prefer using an explicit list"